src/component/Tooltip.tsx

```diff
@@ -52,7 +52,7 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Omi
    * If false, then Tooltip is never displayed.
    * If active is undefined, Recharts will control when the Tooltip displays. This includes mouse and keyboard controls.
    */
-  active?: boolean | undefined;
+  active?: boolean;
```

Comment:
The union was making the storybook render a "Provide Object" UI, instead of a boolean toggle.

The undefined is implied by the question mark.